packages/graphql/tests/tck/connections/projections/create.test.ts

```diff
@@ -132,41 +134,29 @@ describe("Cypher -> Connections -> Projections -> Create", () => {
         });
 
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
-            "CALL {
-            CREATE (this0:Movie)
-            SET this0.title = $this0_title
-            RETURN this0
-            }
-            CALL {
-            CREATE (this1:Movie)
-            SET this1.title = $this1_title
-            RETURN this1
-            }
+            "UNWIND [ { title: $create_param0 }, { title: $create_param1 } ] AS create_var1
```

Comment:
There was an open conversation regarding this, I think that the whole query is more readable using this approach, also, consider that this UNWIND statement is very likely to be a little different from the original GraphQL input, for instance with the `@populatedBy` directive it's very likely that the computed values will be placed as UNWIND parameters.
But as you said nothing prevents us to pass them directly as Cypher parameter.
@darrellwarde @angrykoala extra thoughts?

Reply:
All I know is that "parameterize everything" is the preference from the Cypher planner's point of view - I believe that the planner will have to replan every time without these parameterized. If you think about a GraphQL client application, the query will be hardcoded into a component, so the creation is likely to always be the same or similar, so the queries will be quite optimizable by the planner when they reach the database. I also think more readable in paramaters, because they format nicer!!